src/components/enterprise-user-subsidy/data/utils.js

```diff
@@ -2,9 +2,12 @@
 import { LICENSE_STATUS } from './constants';
 import {
   isDefinedAndNotNull,
-  isNull,
   hasValidStartExpirationDates,
 } from '../../../utils/common';
 
-export const hasValidSubscription = (plan, license) => (hasValidStartExpirationDates(plan) && !isNull(license)
-    && isDefinedAndNotNull(license) && license.status === LICENSE_STATUS.ACTIVATED);
+export const hasValidSubscription = (plan, license) => (
+  isDefinedAndNotNull(plan)
```

Comment:
Adding this `isDefinedAndNotNull(plan)` check since `hasValidStartExpirationDates` will fail if `plan` isn't an object.